src/main/groovy/com/ofg/uptodate/finder/NewVersionFinderInAllRepositories.groovy

```diff
@@ -43,7 +45,8 @@ class NewVersionFinderInAllRepositories {
         if (dependenciesWithNewVersions.isEmpty()) {
             loggerProxy.info(log, NO_NEW_VERSIONS_MESSAGE)
         } else {
-            loggerProxy.warn(log, NEW_VERSIONS_MESSAGE_HEAD + dependenciesWithNewVersions.join('\n'))
+            List<Dependency> sortedUpdates = dependenciesWithNewVersions.sort(new DependencyGroupAndNameComparator())
+            loggerProxy.warn(log, "$NEW_VERSIONS_MESSAGE_HEAD\n${sortedUpdates.join('\n')}")
```

Comment:
warn? maybe [lifecycle](https://gradle.org/docs/current/userguide/logging.html) would be enough?